src/index.ts

```diff
@@ -306,3 +309,28 @@ export function requestDOMPermit(
       })
   );
 }
+
+export const mockDecoyActivationEvents = (generator = 'PL NEWS WEB') => {
+  console.warn(
+    "`mockDecoyActivationEvents()` should only ever be called in development. If you're seeing this in production, please check your code!"
+  );
+
+  function decoyEventMockHandler({ detail }: DecoyEvent) {
+    if (detail.active === true) {
+      window.dispatchEvent(
+        new CustomEvent<DecoyEventDetail>('decoyActive', {
```

Comment:
We can use the `PresentationLayerCustomEvents` enum here, instead of the string

Reply:
Good spot